src/ape/api/providers.py

```diff
@@ -180,7 +180,7 @@ def get_balance(self, address: str) -> int:
         """
 
     @abstractmethod
-    def get_code(self, address: str) -> bytes:
+    def get_code(self, address: AddressType) -> bytes:
         """
         Get the bytes a contract.
 
```

Comment:
The arg doc needs to be updated:

```python
Args:
    address (``AddressType``): The address of the contract.

Returns:
    :class:`~ape.types.ContractCode`: The contract bytecode.
```

Reply:
I think this should be resolved in newest commit.